internal/types/array.go

```diff
@@ -111,3 +111,38 @@ func (a *Array) Append(values ...any) error {
 func (a *Array) RemoveByPath(path Path) {
 	removeByPath(a, path)
 }
+
+// Contains checks if the Array contains the given value.
+func (a *Array) Contains(value any) bool {
+	for _, v := range a.s {
```

Comment:
Why don't use here `slices.Contains`?

Reply:
Ah! This is actually something new to me :) Yeah, I believe I should use it!